src/serverdlgbase.ui

```diff
@@ -60,7 +60,7 @@
       </sizepolicy>
      </property>
      <property name="currentIndex">
-      <number>0</number>
+      <number>1</number>
```

Comment:
What’s this change?

Reply:
I‘m not quite sure since it was changed by the QtDesigner. Not sure if it will break something or not if I manually change it back.

Reply:
It usually happens if you save when exiting the UI Designer having examined a field and not left it in its "clean" state.  I think it should be reverted.

Reply:
@BLumia could you please revert this as Peter suggested?

Reply:
> It usually happens if you save when exiting the UI Designer having examined a field and not left it in its "clean" state.

Yeah as you can see I did add the `extracomment` field with the designer mode. I didn't touch any other thing so it's probably caused by that. anyway here we go :)

```suggestion
      <number>0</number>
```